constants/ModGuiSwitcher.json

```diff
@@ -125,6 +125,15 @@
         "codes.biscuit.skyblockaddons.gui.SkyblockAddonsGui"
       ]
     },
+    "SkyblockTweaks": {
+      "description": [
+        "Auction Stuff!"
+      ],
+      "command": "sbt config",
+      "guiPath": [
+        " mrfast.sbt.config.ConfigGui"
```

Comment:
I feel like that starting space is wrong

Reply:
maybe 👀